__tests__/app/(dashboard)/layout.test.tsx

```diff
@@ -20,10 +22,19 @@ jest.mock("next/navigation", () => ({
 describe('Layout tests',()=>{
     beforeEach(()=>{
         (myGetServerSession as jest.Mock).mockResolvedValue({user:{id: 'foo'}});
-        (fetchRedis as jest.Mock).mockResolvedValue(['foo'])
+        (fetchRedis as jest.Mock).mockResolvedValue(['foo']);
+        (getFriendsById as jest.Mock).mockResolvedValue([{
+            name: 'bob',
+            email: 'test.user@gmail.com',
+            image: 'stub',
+            id: '1701',
+        }]);
+        (SidebarChatList as jest.Mock).mockImplementation(() => {
+            return <ul aria-label='chat list'/>;
```

Comment:
Does this make the actual implementation of the aria label redundant?

Reply:
no, because it's an accessibility thing as well. It's worth it to remember this app actually has to run, not just pass tests